### PR TITLE
[MIG][15.0] hr_recruitment: migration script

### PR DESCRIPTION
**Pull request by sang250399**
This PR
=========
Migration done

### CODE REVIEW

**Review by daonamutc1 — APPROVED** [review]

File: docsource/modules140-150.rst [hunk 1]
```diff
@@ -180,7 +180,7 @@ Module coverage 14.0 -> 15.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | hr_presence                                     |                      |No DB layout changes.                            |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| hr_recruitment                                  |                      |                                                 |
+| hr_recruitment                                  |Done                  |                                                 |
```
Comment (pedrobaeza) [inline docsource/modules140-150.rst:183]:
```suggestion
| hr_recruitment                                  | Done                 |                                                 |
```

File: openupgrade_scripts/scripts/hr_recruitment/15.0.1.1/upgrade_analysis_work.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---Models in module 'hr_recruitment'---
+---Fields in module 'hr_recruitment'---
+hr_recruitment / hr.applicant             / description (text)            : type is now 'html' ('text')
+# DONE: pre-migration: convert text fields to html
+
+hr_recruitment / hr.applicant             / meeting_ids (one2many)        : NEW relation: calendar.event
+# NOTHING TO DO: support count mettings in smart button with compute not store
```
Comment (pedrobaeza) [inline openupgrade_scripts/scripts/hr_recruitment/15.0.1.1/upgrade_analysis_work.txt:7]:
```suggestion
# NOTHING TO DO: the inverse field already exists in v14, and this one is a "virtual" field.
```

**Review by pedrobaeza — APPROVED** [review]
Please take not for other times

**Pull request merged**